static/timeago.js

```diff
@@ -0,0 +1,20 @@
+var timestamp = document.getElementById('time').getAttribute('data-time');
+var ago = document.getElementById('ago');
+
+var sec = ((new Date()).getTime() / 1000) - parseInt(timestamp);
+var min = sec / 60;
+var hour = min / 60;
+var day = hour / 24;
+
+var timeString = '';
+if (day > 1) {
```

Comment:
Shouldn't it be `>=` ?

Reply:
Well, day will only be exactly equal to 1 for about a second, so it's not really necessary. I could make these >= instead of > but the one second difference probably isn't worth it.

Reply:
not 1 second, `30 mins` actually. `Math.round()` will round time `>=23.5` to `24`  😉

Reply:
Suppose the hour turns out to be something `>=23.5` then your logic will display `24 hours ago`